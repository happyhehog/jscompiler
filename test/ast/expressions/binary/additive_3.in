'42' + 3 - x