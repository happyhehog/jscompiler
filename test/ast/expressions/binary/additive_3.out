ProgramNode (1:0, 1:11)
    BinaryExpressionNode (1:0, 1:11) { operator: - }
    --> left operand:
        BinaryExpressionNode (1:0, 1:7) { operator: + }
        --> left operand:
            LiteralNode (1:0, 1:0) : { value: '42' }
        --> right operand:
            LiteralNode (1:7, 1:7) : { value: 3 }
    --> right operand:
        IdentifierNode (1:11, 1:11) : { name: x }
